reactnative/ReactNativeTemplateApp/index.android.js

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
It will probably end up there (along with the index.ios.js that is still in the iOS repo).
I'll get to that when I figure out the template app.